Header header
string name
string message